state/protocol/inmem/epoch.go

```diff
@@ -127,6 +149,10 @@ func (es *setupEpoch) Cluster(_ uint) (protocol.Cluster, error) {
 	return nil, protocol.ErrEpochNotCommitted
 }
 
+func (es *setupEpoch) ClusterByChainID(_ flow.ChainID) (protocol.Cluster, error) {
+	return nil, protocol.ErrEpochNotCommitted
+}
```

Comment:
It returns `protocol.ErrEpochNotCommitted`, because `Cluster` was returning the same error:

```
func (es *setupEpoch) ClusterByChainID(_ flow.ChainID) (protocol.Cluster, error) {
	return nil, protocol.ErrEpochNotCommitted
}
```